index.js

```diff
@@ -231,7 +231,7 @@ function gameOver() {
   audio.hit.play().catch(() => gameOverText.style.display = "initial");
```

Comment:
Could you change `display` property to flex in this line too?

Reply:
@indremak Of course!